snapcraft/meta/snap_yaml.py

```diff
@@ -159,9 +159,9 @@ def write(project: Project, prime_dir: Path, *, arch: str, version: str, grade:
     snap_metadata = SnapMetadata(
         name=project.name,
         title=project.title,
-        version=version,
+        version=project.version,
         summary=project.summary,
-        description=project.description,
+        description=project.description,  # type: ignore
```

Comment:
why is the type ignored?

Reply:
It's because description is now optional in project. At this point we're already sure it has a value, but mypy still thinks it can be None.